src/send/saga.ts

```diff
@@ -264,9 +284,17 @@ export function* sendPaymentSaga({
 
     yield* put(sendPaymentSuccess({ amount, tokenId }))
     SentryTransactionHub.finishTransaction(SentryTransaction.send_payment)
-  } catch (e) {
-    yield* put(showErrorOrFallback(e, ErrorMessages.SEND_PAYMENT_FAILED))
+  } catch (err) {
+    yield* put(showErrorOrFallback(err, ErrorMessages.SEND_PAYMENT_FAILED))
     yield* put(sendPaymentFailure())
```

Comment:
Looks like we wouldn't want the error message if the pin was cancelled.

And maybe not the `sendPaymentFailure()`? Unless it's used for switching the button state back to enabled.

Reply:
The showErrorOrFallback will show a pin required error message, and `sendPaymentFailure` just resets the `isSending` state so should be fine to include.
I was confused by this as well originally 😄. left a couple of comments

Reply:
ok, I guess what I'm suggesting is we change the UX to not display an error message if the pin was cancelled, and not proceed. It's slightly annoying to see the error in that case, since the user decided to cancel.
We did that for swap. https://github.com/valora-inc/wallet/blob/fb39d9985490b4cb7cfa09f559710d193539db06/src/swap/saga.ts#L378-L383

Reply:
makes sense, I think there are a few differences with how send and swap UX works, like for swap we navigate immediately after submitting the tx to the chain but we block on the confirmation screen for send. Would be good to address all that in a separate PR with all the UX changes.